src/components/Article.jsx

```diff
@@ -0,0 +1,56 @@
+import "../styles/article.css"
+import { useEffect, useState } from 'react'
+import { RingLoader } from "react-spinners"
+import { useParams } from "react-router-dom"
+import { format } from "date-fns";
+import getArticles from "../APIs/getArticleAPI"
+
+
+export default function Article() {
+    const [article, setArticle] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const [hasError, setHasError] = useState(false)
+
+    const {article_id} = useParams()
+
+    useEffect(() => {
+        setIsLoading(true)
+        setHasError(false)
+        getArticles(`${article_id}`)
```

Comment:
I think article_id is already a string - so it looks like you're taking a string and putting it inside a string literal to make it a string 😄 might be wrong though!